sources/models/shared-defaults/kubernetes-metal.toml

```
@@ -0,0 +1,19 @@
+[settings.kubernetes]
+cluster-domain = "cluster.local"
+standalone-mode = false
+authentication-mode = "tls"
+pod-infra-container-image = "k8s.gcr.io/pause:3.2"
```

We should switch these over to the ECR public images:
```suggestion
pod-infra-container-image = "public.ecr.aws/eks-distro/kubernetes/pause:3.5"
```